hbase_storage/hbase1.0.0/src/main/java/com/splicemachine/access/client/MemStoreFlushAwareScanner.java

```diff
@@ -69,6 +70,7 @@ public MemStoreFlushAwareScanner(HRegion region, Store store, ScanInfo scanInfo,
 			this.memstoreAware = memstoreAware;
 			this.initialValue = initialValue;
 			this.region = region;
+			this.stopRow = Bytes.equals(scan.getStopRow(), HConstants.EMPTY_END_ROW) ? null : scan.getStopRow();
 		}
```

Comment:
Why for hbase1.0.0, we don't check "isStopRow(peek)" in addition to the check of "peek==null in peek()"?

Reply:
I missed that one, good eye!